### DISCUSSION TITLE
Dynamic Update v4 Script Gone?

### DISCUSSION THREAD

**Discussion by UdderlyEvelyn** [AveYo/MediaCreationTool.bat#40] [General]
The v4 script is just a dead link in the readme, the archived folder has v3 which worked but I thought that was odd. As well, after applying it windows update still reported incompatibility, not sure if that might have required a reboot though. I just went with mounting the iso. Thanks for this, btw.

**Comment by AveYo** [comment]
There's a [wip] somewhere in there "work-in-progress" meaning it's not released yet 

I just did not want to "sleep on it" for days/weeks/.. like I usually do until I personally like the result, 
and instead put up an update for the other improved parts that are significant enough.

I've hit an unwanted behavior with v4 so I'm holding it off for now, thanks for your patience

**Comment by AveYo** [comment]
Now it's up (but not what I had in mind - I had to fallback to the previous method with some small improvements).
The revised method is still on my radar, and will share it if and when I get it sorted.

**Reply by AveYo** [reply]
The revised method is the current one, **v7** - introduced with v6 & v6f
Make sure you use the latest script.  
It should be more future-proof, but only time will tell.
